### DISCUSSION TITLE
Frigate 0.12 (Standalone - not HA) - areas of object detection

### DISCUSSION THREAD

**Discussion by craigcurtin-dev** [blakeblackshear/frigate#7907] [General]
Guys, New user and trying to streamline the system - it has been running fine for two weeks now and i am trying to streamline. We live in a windy area and am getting some inaccurate events notified. Does the Yellow Star that comes up in the event list signify which part of the image triggered the event - or is there somewhere to see this so i can look for recurring patterns and then mask out those areas ?

If it makes a difference i am using mainly Dahua TIOC cameras and a Coral USB TPU 

regards

Craig

**Comment by NickM-27** [comment]
> Does the Yellow Star that comes up in the event list signify which part of the image triggered the event

the yellow star is just a button to favorite the event (keep it from being cleaned up automatically)

generally if you have a lot of false events and it isn't clear what caused the motion you would want to watch the debug live view with motion boxes enabled

**Reply by craigcurtin-dev** [reply]
Oh OK thanks for that feedback. Would seem to be a good idea to add something like this - if the image part that triggered the event could be highlighted so it could be easily then masked out without having to guess ?

Craig
